packages/entity-secondary-cache-local-memory/src/LocalMemorySecondaryEntityCache.ts

```diff
@@ -0,0 +1,17 @@
+import { GenericSecondaryEntityCache } from '@expo/entity';
+import {
+  GenericLocalMemoryCacher,
+  LocalMemoryCache,
+} from '@expo/entity-cache-adapter-local-memory';
+
+/**
+ * A local memory {@link GenericSecondaryEntityCache}.
+ */
+export default class LocalMemorySecondaryEntityCache<
+  TFields,
+  TLoadParams
+> extends GenericSecondaryEntityCache<TFields, TLoadParams> {
+  constructor(localMemoryCache: LocalMemoryCache<TFields>) {
+    super(new GenericLocalMemoryCacher(localMemoryCache), (params) => JSON.stringify(params));
```

Comment:
I suppose it is possible for `TLoadParams` to contain values that don't stringify well (ie) `[Object object]`. That being said, we do also [make assumptions in the RedisCacheAdapter](https://github.com/expo/entity/blob/master/packages/entity-cache-adapter-redis/src/RedisCacheAdapter.ts#L122) that generics like FieldValue can be coerced to a String. Maybe a good idea to document this in the README or function signature, especially if we document something similar for FieldValue